argoverse/utils/se3.py

```diff
@@ -55,20 +55,33 @@ def inverse(self) -> "SE3":
         """
         return SE3(rotation=self.rotation.T, translation=self.rotation.T.dot(-self.translation))
 
-    def right_multiply_with_se3(self, right_se3: "SE3") -> "SE3":
-        """Right multiply the transformation matrix with another SE3 instance.
+    def compose(self, right_se3: "SE3") -> "SE3":
+        """Compose (right multiply) this class' transformation matrix T with another SE3 instance.
 
         Algebraic representation: chained_se3 = T * right_se3
 
         Args:
-            right_se3: instance of SE3 class
+            right_se3: another instance of SE3 class
 
         Returns:
-            chained_se3: instance of SE3 class
+            chained_se3: new instance of SE3 class
         """
         chained_transform_matrix = self.transform_matrix.dot(right_se3.transform_matrix)
```

Comment:
I know this wasn't changed with this PR, but any aversion to using the matrix multiplication operator introduced in Python 3.5 `self.transform_matrix @ right_se3.transform_matrix`? I think it's a bit more readable, especially when following algebraic notation.

Reply:
This is a nice suggestion, thanks. Done